concepticondata/conceptlists/Nagano-2013-1256.tsv

```diff
@@ -0,0 +1,1257 @@
+ID	NUMBER	SRCID	ENGLISH	CONCEPTICON_ID	CONCEPTICON_GLOSS	COMMENT
+Nagano-2013-1256-1	1	0003	head	1256	HEAD	
+Nagano-2013-1256-2	2	0004	have a headache (v)			
+Nagano-2013-1256-3	3	0005	stubborn/People say he does not listen.			
+Nagano-2013-1256-4	4	0007	hair of the head	2678	HAIR OR FUR	
+Nagano-2013-1256-5	5	0009	brain	1583	BRAIN	
+Nagano-2013-1256-6	6	0011	forehead	123	FOREHEAD	
+Nagano-2013-1256-7	7	0013	eye	1248	EYE	
+Nagano-2013-1256-8	8	0014	eyeball			
+Nagano-2013-1256-9	9	0016	eyebrow	1173	EYEBROW	
+Nagano-2013-1256-10	10	0017	eyelash	1540	EYELASH	
+Nagano-2013-1256-11	11	0019	tears	1651	TEAR (OF EYE)	
+Nagano-2013-1256-12	12	0021	blind	1653	BLIND	
+Nagano-2013-1256-13	13	0022	blind	1653	BLIND	
+Nagano-2013-1256-14	14	0024	nose	1221	NOSE	
+Nagano-2013-1256-15	15	0025	snivel	208	NASAL MUCUS (SNOT)	
+Nagano-2013-1256-16	16	0027	ear	1247	EAR	
+Nagano-2013-1256-17	17	0028	ear wax	32	EARWAX	
+Nagano-2013-1256-18	18	0030	deaf	996	DEAF	
+Nagano-2013-1256-19	19	0032	mouth	674	MOUTH	
+Nagano-2013-1256-20	20	0034	lips	478	LIP	
+Nagano-2013-1256-21	21	0036	tongue	1205	TONGUE	
+Nagano-2013-1256-22	22	0038	dumb	1492	DUMB	
+Nagano-2013-1256-23	23	0040	tooth	1380	TOOTH	
+Nagano-2013-1256-24	24	0041	decay (v)			
```

Comment:
We would have 2907 ROT 'To be exposed to decomposition'.

Reply:
It would be a bit too general, I guess, as @MacyL consulted the list and told me this is specifically for teeth, while out ROT is more in the sense of "to wither". Should I map it nonetheless?

Reply:
Gotcha, better leave it unlinked then.